examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -628,13 +628,18 @@ class AppCallbacks : public AppDelegate
 
 AppCallbacks sCallbacks;
 
+constexpr EndpointId kNetworkCommissioningEndpointSecondary = 0xFFFE;
+
 } // namespace
 
 static void InitServer(intptr_t context)
 {
     // Init ZCL Data Model and CHIP App Server
     chip::Server::GetInstance().Init(&sCallbacks);
 
+    // We only have network commissioning on endpoint 0.
+    emberAfEndpointEnableDisable(kNetworkCommissioningEndpointSecondary, false);
```

Comment:
Is this being done in this particular "flavor" of the all-clusters-app because the zap config is shared between all the "flavors"?

i.e there might be devices running this application that might have other network commissioning cluster instances?

Reply:
Yes, exactly.  The Linux versions supports both WiFi and Thread in some configurations.